apps/recovery-utility/renderer/context/Workspace.tsx

```diff
@@ -65,7 +66,14 @@ export const WorkspaceProvider = ({ children }: Props) => {
     app: 'utility',
     relayBaseUrl,
     deriveWallet: (input) => {
-      const nativeAssetId = (getAssetConfig(input.assetId)?.nativeAsset ?? input.assetId) as keyof typeof WalletClasses;
+      let transferableToken = false;
+      const config = getAssetConfig(input.assetId);
+      if (config?.address && isTransferableToken(input.assetId)) {
```

Comment:
`config?.address` is already done as part of isTransferableToken, remove it please